### DISCUSSION TITLE
proyecto

### DISCUSSION THREAD

**Discussion by jesusvlc** [iptv-org/community#499] [Q&A] [answered]
se ha paralizado este proyecto?

**Comment by freearhey** [comment, marked as answer]
no

**Reply by jesusvlc** [reply]
aah como e pedido que se arregle un canal y antes lo hacian ensegida

**Reply by freearhey** [reply]
unfortunately we have no control over the quality of broadcasts and cannot guarantee that the links will work all the time. but as soon as there is a working link we will definitely add it to the playlist.

**Reply by jesusvlc** [reply]
https://m.youtube.com/watch?v=WLf_fUhD7ak&embeds_referring_euri=https%3A%2F%2Fcanalred.tv%2F&embeds_referring_origin=https%3A%2F%2Fcanalred.tv&source_ve_path=MjM4NTE&feature=emb_title , este canal ya va 24horas es de españa. Y en el enlace que esta ahora en el proyecto no funciona

**Reply by freearhey** [reply]
i've created a new request (https://github.com/iptv-org/iptv/issues/14496). the link will appear in the playlist as soon as other team members check it out

**Reply by jesusvlc** [reply]
muchas gracias
